napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -78,350 +91,26 @@ class QtImageControls(QtBaseImageControls):
-        self.renderLabel = QLabel(trans._('rendering:'))
-
-        self.depictionComboBox = QComboBox(self)
-        depiction_options = [d.value for d in VolumeDepiction]
-        self.depictionComboBox.addItems(depiction_options)
-        index = self.depictionComboBox.findText(
-            self.layer.depiction, Qt.MatchFlag.MatchFixedString
-        )
-        self.depictionComboBox.setCurrentIndex(index)
-        self.depictionComboBox.currentTextChanged.connect(self.changeDepiction)
-        self.depictionLabel = QLabel(trans._('depiction:'))
-
-        # plane controls
-        self.planeNormalButtons = PlaneNormalButtons(self)
-        self.planeNormalLabel = QLabel(trans._('plane normal:'))
-        action_manager.bind_button(
-            'napari:orient_plane_normal_along_z',
-            self.planeNormalButtons.zButton,
-        )
-        action_manager.bind_button(
-            'napari:orient_plane_normal_along_y',
-            self.planeNormalButtons.yButton,
-        )
-        action_manager.bind_button(
-            'napari:orient_plane_normal_along_x',
-            self.planeNormalButtons.xButton,
-        )
-        action_manager.bind_button(
-            'napari:orient_plane_normal_along_view_direction_no_gen',
-            self.planeNormalButtons.obliqueButton,
-        )
-
-        self.planeThicknessSlider = QLabeledDoubleSlider(
-            Qt.Orientation.Horizontal, self
-        )
-        self.planeThicknessLabel = QLabel(trans._('plane thickness:'))
-        self.planeThicknessSlider.setFocusPolicy(Qt.NoFocus)
-        self.planeThicknessSlider.setMinimum(1)
-        self.planeThicknessSlider.setMaximum(50)
-        self.planeThicknessSlider.setValue(self.layer.plane.thickness)
-        self.planeThicknessSlider.valueChanged.connect(
-            self.changePlaneThickness
-        )
-
-        sld = QDoubleSlider(Qt.Orientation.Horizontal, parent=self)
-        sld.setFocusPolicy(Qt.FocusPolicy.NoFocus)
-        cmin, cmax = self.layer.contrast_limits_range
-        sld.setMinimum(cmin)
-        sld.setMaximum(cmax)
-        sld.setValue(self.layer.iso_threshold)
-        sld.valueChanged.connect(self.changeIsoThreshold)
-        self.isoThresholdSlider = sld
-        self.isoThresholdLabel = QLabel(trans._('iso threshold:'))
-
-        sld = QSlider(Qt.Orientation.Horizontal, parent=self)
-        sld.setFocusPolicy(Qt.FocusPolicy.NoFocus)
-        sld.setMinimum(0)
-        sld.setMaximum(100)
-        sld.setSingleStep(1)
-        sld.setValue(int(self.layer.attenuation * 200))
-        sld.valueChanged.connect(self.changeAttenuation)
-        self.attenuationSlider = sld
-        self.attenuationLabel = QLabel(trans._('attenuation:'))
+        self._add_widget_controls(self._interpolationControl)
+        self._depictionControl = QtDepictionControl(self, layer)
+        self._add_widget_controls(self._depictionControl)
+        self._renderControl = QtImageRenderControl(self, layer)
```

Comment:
```suggestion
        self._render_control = QtImageRenderControl(self, layer)
```

We should use proper naming convention (may require update in multiple places)